libraries/AP_Notify/Buzzer.cpp

```diff
@@ -22,9 +22,14 @@
 #include "AP_Notify.h"
 
 #ifndef HAL_BUZZER_ON
+#if !defined(HAL_BUZZER_PIN)
+#define HAL_BUZZER_ON (pNotify->get_buzz_level())
+#define HAL_BUZZER_OFF (!pNotify->get_buzz_level())
+#else
```

Comment:
Please indent this.

It's a bit doubtful if HAL_BUZZER_ON/OFF should depend on HAL_BUZZER_PIN given that HALs/boards can set (or not) the HAL_BUZZER_ON. The current logic says that if board has defined a pin for buzzer then it also has chosen the logic level - I'm not sure one does necessarily imply the other.

Reply:
it's not an unreasonable requirement though

Reply:
Indented the blocks.

Leaving the requirement on logic level.